internalsdk/android_test.go

```diff
@@ -94,6 +93,10 @@ func TestProxying(t *testing.T) {
 		newResult, err := Start("testapp", "en_US", testSettings{}, testSession{})
 		require.NoError(t, err, "Should have been able to start lantern twice")
 		require.Equal(t, result.HTTPAddr, newResult.HTTPAddr, "2nd start should have resulted in the same address")
+
+		// give the client time to initialize. This helps with consistency in tests
+		time.Sleep(1 * time.Second)
```

Comment:
Is there something you can expose to synchronize on that will guarantee that it's initialized? Sleeping in tests is always a bad idea and the tests are already super flakey.

Reply:
Removed. `Start` actually blocks until it's ready.